systemtest/driver/inifile.py

```diff
@@ -37,3 +38,27 @@ def instanciate_tempalte(*, src: pathlib.Path, args: Dict, dest: pathlib.Path):
     tpl = jinja2.Template(src.read_text())
     content = tpl.render(args)
     dest.write_text(content)
+
+
+class TrafficConstraint(NamedTuple):
+    door_id: int
+    outflow: int
+    max_agents: int
+
+
+def parse_traffic_constraints(inifile: pathlib.Path):
+    tree = ET.parse(inifile)
+    root = tree.getroot()
+    traffic_constraints = {}
+    for tc in root.iter("traffic_constraints"):
+        for door in tc.iter("door"):
+            traffic_constraints[
+                int(door.attrib["trans_id"])
+            ] = TrafficConstraint(
+                int(door.attrib["trans_id"]),
+                int(door.get("outflow", -1)),
+                int(door.get("max_agents", -1)),
+            )
+
+    assert traffic_constraints, "Could not parse traffic constraints"
```

Comment:
I strongly prefer if only tests or 'check' functions assert. Imo this should either just return 'None' or throw an exception.

Reply:
done